src/App.js

```diff
@@ -536,6 +538,21 @@ export default class App extends Component {
                           </AuthorizedElement>
                         }
                       />
+                      <Route
+                        path="Entities"
+                        element={
+                          <AuthorizedElement tokenDecoded={this.state.tokenData} iSuperAdmin={true}>
```

Comment:
also not admin should be able to see entities (then which ones it's another matter, and it should be tackled by the API)